assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -40,10 +40,15 @@ import PageHeading from './pageHeading';
 import NoResults from './noResults';
 
 const FilterContainer = styled.div`
-  padding: 0 20px 20px;
+  padding: 0 20px 20px 0;
+  margin: 0 20px;
   border-bottom: ${({ theme: t }) => t.subNavigationBar.border};
 `;
 
+const ListBarContainer = styled.div`
+  margin: 10px 0 0 20px;
+`;
+
```

Comment:
this fixes the dashboard view alignment, trying to fix the grid without fixing this was making my eyes dizzy!  Moved the margin for the ListBar to a container so that the listbar isn't in charge of its placement on the page and can more readily get reused without setting space.